flixel/addons/text/FlxTypeText.hx

```diff
@@ -516,6 +521,7 @@ class FlxTypeText extends FlxText
 			if (_length >= _finalText.length && _typing && !_waiting && !_erasing)
 			{
 				onComplete();
+				onCharacterTyped.dispatch();
```

Comment:
Seems odd to dispatch this after onComplete. I also wonder if rather than calling it in two places can we just dispatch it once, [immediately after the text is changed](https://github.com/HaxeFlixel/flixel-addons/blob/60c5991aa94d73ecb7a57629375d19ed1b98a954/flixel/addons/text/FlxTypeText.hx#L518)